lib/files.js

```diff
@@ -0,0 +1,54 @@
+const { checkIfPathMatches } = require('./parseOwners')
+
+/**
+ * process the PR based on dependent settings
```

Comment:
Probably want to update the description here.

Reply:
done